frontend/src/routes/SearchPage/searchDefinitions.tsx

```diff
@@ -1360,10 +1360,11 @@ export function CreateDetailsLink(item: any) {
     switch (item.kind) {
         case 'cluster':
             return <a href={`/multicloud/clusters/${item.name}/overview`}>{item.name}</a>
+
         case 'application':
-            if (item.apigroup === 'app.k8s.io') {
+            if (item.apigroup === 'app.k8s.io' || item.apigroup === 'argoproj.io') {
                 // only redirect to apps page if it is an ACM application
-                return <a href={`/multicloud/applications/${item.namespace}/${item.name}`}>{item.name}</a>
+                return <a href={`/multicloud/applications/${item.namespace}/${item.name}?apiVersion=argoproj.io%${item.cluster}`}>{item.name}</a>
```

Comment:
apiVersion could be either `app.k8s.io` or `argoproj.io`
```suggestion
                return <a href={`/multicloud/applications/${item.namespace}/${item.name}?apiVersion=${item.apigroup}%${item.cluster}`}>{item.name}</a>
```

Reply:
Actually, you'll also need to add this change:
```suggestion
                return <a href={`/multicloud/applications/${item.namespace}/${item.name}?apiVersion=${item.apigroup}%2F${item.apiversion}$cluster=${item.cluster}`}>{item.name}</a>
```